repository/asset_repository.go

```diff
@@ -0,0 +1,43 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/vkhichar/asset-management/domain"
+)
+
+const (
+	GetAssetDetails = "SELECT id,category,status,purchase_at,purchase_cost,name,specifications FROM assets"
```

Comment:
this constant is not used outside this package, so this should be un exported

Reply:
okay I got it!